tests/test_dialects.py

```diff
@@ -801,13 +801,13 @@ def test_presto(self):
         )
         self.validate(
             "DATE_PARSE(x, '%Y-%m-%d %H:%i:%s')",
-            "FROM_UNIXTIME(UNIX_TIMESTAMP(x))",
+            "DATE_FORMAT(x, 'yyyy-MM-dd HH:mm:ss')",
             read="presto",
             write="hive",
         )
         self.validate(
             "DATE_PARSE(x, '%Y-%m-%d')",
-            "FROM_UNIXTIME(UNIX_TIMESTAMP(x, 'yyyy-MM-dd'))",
+            "DATE_FORMAT(x, 'yyyy-MM-dd HH:mm:ss')",
```

Comment:
Is this right?

Reply:
This isn't right. `DATE_PARSE` takes a string and returns a timestamp while `DATE_FORMAT` takes a timestamp and returns a string.